engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -890,4 +880,14 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
```

Comment:
@geoffjentry That's how it is currently, but then going back to @kshakir 's comments that we should not be using `CromwellServer`'s reference directly to get that actor, (and instead plumbing it's reference down to the actors needing it), how do you see, for instance, the `WorkflowActor` getting hold of the `MaterialieWorkflowDescriptorActor` reference to materialize a workflow from sources?